packaging/crio.conf.d/microshift-ovn.conf

```diff
@@ -3,7 +3,7 @@
 # by declaring this CRI-O will wait until that network is configured.
 cni_default_network = "ovn-kubernetes"
 
-[crio.runtime.workloads.management]
-activation_annotation = "target.workload.openshift.io/management"
-annotation_prefix = "resources.workload.openshift.io"
-resources = { "cpushares" = 0, "cpuset" = "0" }
+# [crio.runtime.workloads.management]
```

Comment:
We've had some other conversations about making these settings configurable or installable separately. If this PR works, we should talk about the long term approach to handling that.